ChangeLog.md

```diff
@@ -16,12 +16,16 @@
  - Make `substitute` more powerful in the UI by allowing the substitution of multiple expressions simultaneously
  - Make `def` and `sub` in `CommandUI` now automatically perform a `minimalSimplify`
  - Add support for `_` in `evaluate` (used as `eval x^2 x=_` when `_` is a  `Double`)
+ - Add `version` command to keyword interface
```

Comment:
Make this conform to other changelog messages that say `CommandUI` by either making this say "CommandUI" or making them say `keyword interface`